src/bluesky/consolidators.py

```diff
@@ -271,21 +272,22 @@ def adapter_parameters(self) -> Dict:
         return {"dataset": self._sres_parameters["dataset"].strip("/").split("/"), "swmr": self.swmr}
 
 
-class TIFFConsolidator(ConsolidatorBase):
-    supported_mimetypes = {"multipart/related;type=image/tiff"}
-
-    def __init__(self, stream_resource: StreamResource, descriptor: EventDescriptor):
+class MultipartRelatedConsolidator(ConsolidatorBase):
+    def __init__(self, permitted_extensions: str, stream_resource: StreamResource, descriptor: EventDescriptor):
```

Comment:
shouldn't `permitted_extensions` be an iterable (list) of str?

Reply:
Yep, originally had it as a string but obviously changed to list since we can allow multiple. Actually, I guess a Set makes more sensr than list here anyway?

Reply:
yes, set is perfect!